pkg/application/interfaces.go

```diff
@@ -98,20 +98,27 @@ type remoteDoguRegistry interface {
 type maintenanceMode interface {
 	domainservice.MaintenanceMode
 }
-type globalConfigEntryRepository interface {
-	domainservice.GlobalConfigEntryRepository
+
+type globalConfigRepository interface {
+	domainservice.GlobalConfigRepository
 }
 
+// TODO: remove this when refactoring is done
 type doguConfigEntryRepository interface {
 	domainservice.DoguConfigEntryRepository
 }
 
+type doguConfigRepository interface {
+	domainservice.DoguConfigRepository
+}
+
+// TODO: remove this when refactoring is done
 type sensitiveDoguConfigEntryRepository interface {
 	domainservice.SensitiveDoguConfigEntryRepository
 }
 
-type configEncryptionAdapter interface {
-	domainservice.ConfigEncryptionAdapter
+type sensitiveDoguConfigRepository interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `sensitiveDoguConfigRepository` is unused
<!-- __reviewdog__:ChA3NmZkOTNkYzc3Nzc0ZTQxEgpjaGVja3N0eWxl -->